handbook/engineering/search/index.md

```diff
@@ -47,15 +50,20 @@ Our current focus is documented in the [tracking issue for the current milestone
   - **Outcomes:** OpenGrok users eagerly migrate to Sourcegraph, and can run complex searches on Sourcegraph.
   - **Owner:** Rijnard
   - **Status:** In progress
+  - **Estimated completion:** TODO
 - **Improve syntax for existing filters like `repohasfile`**
   - **Problem:** Creating a custom filter name for every permutation of search use does not scale (existing filter examples: `repohasfile`, `hascommitafter`).
   - **Outcomes:** We have an expressive syntax that scales and effectively leverages existing filters to achieve searches that previously would have required a custom/new filter.
+  - **Tentative Plan:** Implement a query language extension to express relations (i.e., rules) on code, files, repositories, and commits
   - **Owner:** Rijnard/Stefan
   - **Status:** Not started
+  - **Estimated start:** TODO
+  - **Estimated effort:** TODO
 - **Revision search**
   - **Problem:** Users want to search across branches with the same ease as searching across repositories, and are currently unable to do so.
   - **Owner:** Stefan
   - **Status:** In progress
+  - **Estimated completion**: TODO
```

Comment:
@stefanhengl given you recently started focusing on performance, will you be able to keep making progress on this? If so, can you add an (even rough) estimated completion milestone?

Reply:
This is heavily dependent on @rvantonder work to support expressions for filters. We already introduced a separate filter `rev:` (equivalent to `@` but separated from `repo:`) and documented the previously undocumented revision syntax. Once we have expressions for filters, `rev:` unfolds its full power in queries like this one `repo:foo (rev:bar or rev:baz)`. Given the focus on performance, I don't plan to work on this apart from bug-fixes.

Reply:
Thank you for the clarification 🙂 Beyond adding expressions for filters, what would be next steps when we resume this work?

Reply:
This functionality is subsumed by https://github.com/sourcegraph/sourcegraph/issues/13126. It's in progress and will go from feature-flagged to release in the next two iterations. I've made edits to this PR to reflect this, let's carry on the conversation on those changes :-)